QueAns/tests.py

```diff
@@ -0,0 +1,16 @@
+#couldn't test it
+from django.test import Client
+
+c = Client()
+response = c.post('/upvote/',{'user_id':'prakhar_96','question_id':'1'})
+self.assertEqual(response.status_code, 200)
+
+response = c.post('/downvote',{'user_id':'prakhar_96','question_id':'1',})
+self.assertEqual(response.status_code,200)
```

Comment:
We should also assert things like what was returned from the server, what was response data vs just status code.